scripts/add_xe_vdw.py

```diff
@@ -0,0 +1,30 @@
+from openff.toolkit import ForceField
+from openff.toolkit.typing.engines.smirnoff import vdWHandler
+from openff.units import unit
+
+sage_21 = ForceField("openff-2.1.0.offxml")
+sage_21_uc = ForceField("openff_unconstrained-2.1.0.offxml")
+
+# Add the new parameters, from:
+# Tang, K.T., Toennies, J.P. New combining rules for well parameters and shapes
+# of the van der Waals potential of mixed rare gas systems. Z Phys D - Atoms,
+# Molecules and Clusters 1, 91–101 (1986). https://doi.org/10.1007/BF01384663
+xe_param = vdWHandler.vdWType(
+    smirks="[#54:1]",
+    epsilon=0.56108 * unit.kilocalorie / unit.mole,
+    sigma=4.363 * unit.angstrom,
+    id="n36",
+)
+
+xe_idx_sage21 = len(sage_21.get_parameter_handler("vdW").parameters) + 1
+sage_21.get_parameter_handler("vdW").parameters.insert(xe_idx_sage21, xe_param)
+sage_21.to_file("../openforcefields/offxml/openff-2.1.1.offxml")
+sage_21_uc.get_parameter_handler("vdW").parameters.insert(xe_idx_sage21, xe_param)
+sage_21_uc.to_file("../openforcefields/offxml/openff_unconstrained-2.1.1.offxml")
+
+# Make sure new FF loads with most recent toolkit version
+test_sage_211 = ForceField("../openforcefields/offxml/openff-2.1.1.offxml")
+test_sage_211_uc = ForceField(
+    "../openforcefields/offxml/openff_unconstrained-2.1.1.offxml"
+)
```

Comment:
A suggestion for @j-wags's consideration is whether or not it's useful to further test that this can create system(s) and produce sensible results. I don't see why our infrastructure would treat Xe any differently than a monoatomic molecule - but that's the point of tests

```python
from openff.interchange.drivers.all import get_openmm_energies, get_amber_energies, ...

topology = Topology.from_molecules(
    [
        Molecule.from_smiles("[Xe]"),
        Molecule.from_smiles("[Xe]"),
    ]
)

# Place them a few (4 or so?) A apart
topology.set_positions = Quantity(
    ...
)

interchange = test_sage_211.create_interchange(topology)

get_openmm_energies(...).compare(get_amber_energies(...))
```

Reply:
Great call. Added. This showed me that there's we also need to add a Xe librarycharge.